pilz_robot_programming/src/pilz_robot_programming/robot.py

```diff
@@ -154,6 +156,32 @@ def _robot_commander(self):
     def _robot_commander(self, robot_commander):
         self.__robot_commander = robot_commander
 
+    @property
+    def global_motion_factor(self):
+        """ Defines a factor applied to all commands.
+
+        Both velocity and acceleration scaling are factorized. The command itself remains untouched.
+
+        :getter: Returns the global scaling factor
+
+        :setter: The factor applied to all commands before executing them.
+
+          :raises ValueError: Ff the value is not within (0.0, 1.0]
```

Comment:
typo: Ff -> if

Reply:
What happens if I choose a very small factor, like 1e-16?

Reply:
You move really slow... :snail:

Reply:
Typo resolved.